apps/mudita-center-e2e/wdio.conf.ts

```diff
@@ -115,10 +91,9 @@ export const config: Options.Testrunner = {
       // maxInstances: 5,
       //
       browserName: "chrome",
-      // acceptInsecureCerts: true,
       "goog:chromeOptions": {
         binary: process.env.TEST_BINARY_PATH,
-        args: ["--headless", "--no-sandbox"],
+        args: [],
```

Comment:
what about other arguments? Task is indicating that none of the arguments are used. --headless is understandable, but what about --no-sandbox and others?

Reply:
You don't need the --no-sandbox and other arguments to run the tests locally or on CI/CD, application. Do you have another use case where you need to pass args? Those you're pointing out are not needed to properly run the tests.